src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -119,7 +120,7 @@ internal static IArgument CreateArgumentOperation(ArgumentKind kind, IParameterS
                     //TODO: https://github.com/dotnet/roslyn/issues/18722
```

Comment:
Does this `TODO` mean we'll still need to fix the original crash when #18722 is tackled?

Instead of working around the issue, can we fix the crash in `GetDefaultParameterValue`? Perhaps with:
```
                TypeSymbol constantType = defaultConstantValue.SpecialType == SpecialType.None ?
                    parameterType.GetNullableUnderlyingType() :
                    compilation.GetSpecialType(defaultConstantValue.SpecialType);
```